crates/conformance-test-utils/src/hive/genesis.rs

```diff
@@ -23,6 +36,125 @@ impl HiveGenesisConfig {
     }
 }
 
+/// Convert Hive Genesis Config to Madara Genesis Config
+///
+/// This function will:
+/// 1. Load the Madara genesis file
+/// 2. Construct a Starknet test sequencer
+/// 3. Declare Kakarot contracts
+/// 4. Add Kakarot contracts to Loader
+/// 5. Add Hive accounts to Loader
+/// 6. Serialize Loader to Madara genesis file
+pub async fn serialize_hive_to_madara_genesis_config(hive_genesis: HiveGenesisConfig) -> Result<(), IoError> {
```

Comment:
since the function takes a `hive_genesis`, it could take a `madara_genesis` as well in the form of the loader. This would remove the hardcoded path we have.

Reply:
Gotcha, I think that's a good idea, I'll remove the hardcoded paths and set them as parameters instead 👍